static/css/v3/learn-page.css

```
@@ -1,128 +1,126 @@
+  color: var(--color-text-secondary);
 
-    /* Sans/Desktop/Regular/Large/Loose */
-    font-size: var(--font-size-large);
+  /* Sans/Desktop/Regular/Large/Loose */
+  font-size: var(--font-size-large);
 
-    font-weight: var(--font-weight-regular);
-    line-height: var(--line-height-loose-alt);
-    /* 31.92px */
-    letter-spacing: -0.24px;
+  font-weight: var(--font-weight-regular);
```

Stylelint (17.14.0)

[error] Expected no empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)